CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contribution guidelines
+
+Ongoing development on this project is managed by this project board: https://github.com/neontribe/www/projects/6.
+
+## Getting started on development
+
+Any issues in the `To do` column are assumed to be important enough and have been prioritised.
+
+When contributing to this project please do the following:
+
+1. Pick something near the top of the to do column, which you feel you have the time and confidence to tackle.
+   - We expect you to learn while tackling issues, please don't be put off by a tasks' seeming complexity.
+   - If you need more information please follow up with the person who spawned the task.
+   - Check the [Definition of done](#definition-of-done) for the ticket
+1. Assign yourself and move the ticket to the In progress column.
+1. Create a branch with a name to help match it to the ticket, e.g.: `307-contrib-docs`. _n.b._ [avoid slashes in branch names](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name/2527452#2527452)
+1. Work on the feature branch, make small commits which include the _reason_ for each commit's change.
```

Comment:
Additionally, I like these: https://chris.beams.io/posts/git-commit/

Probably not necessary to include that specific link but the most important one for me is:
https://chris.beams.io/posts/git-commit/#imperative